server/game/cards/09.4-CToH/RegalBearing.js

```diff
@@ -0,0 +1,32 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+
+class RegalBearing extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Drop your bid to 1 and then draw cards equal to the difference between honor dials',
```

Comment:
Yeah, too long.  We really need some idea of what is too long and what isn't - I'm just pointing it out because @jeremylarner as pointed it out to me in the past.